test/infrastructure/docker/internal/docker/util.go

```diff
@@ -27,10 +27,22 @@ import (
 	"sigs.k8s.io/cluster-api/test/infrastructure/docker/internal/docker/types"
 )
 
-const clusterLabelKey = "io.x-k8s.kind.cluster"
-const nodeRoleLabelKey = "io.x-k8s.kind.role"
-const filterLabel = "label"
-const filterName = "name"
+const (
+	clusterLabelKey  = "io.x-k8s.kind.cluster"
+	nodeRoleLabelKey = "io.x-k8s.kind.role"
+	filterLabel      = "label"
+	filterName       = "name"
+
+	failureDomainLabelKey = "io.x-k8s.cluster.x-k8s.failureDomain"
```

Comment:
```suggestion
	failureDomainLabelKey = "io.x-k8s.cluster.failureDomain"
```
nit. Should this be like this?

Previous version looked redundant. I wonder why that is all inverted :)

Reply:
fixed